### ISSUE TITLE
Unable to copy note in the Order notes section

### ISSUE THREAD

**Issue by maximmai** [woocommerce/woocommerce-ios#4019]
**Describe the bug**
In our system, an invoice link is attached to each order, in the note section of an order. However, not only that the link is clickable on the order page on iOS, nor it's copyable to paste in a browser to open. T

**To Reproduce**
1. Go to Woocommerce -> Orders -> Any Order -> Order notes -> Add note  -> any URL" -> Add
2. Go to the iOS app -> Orders -> Order
3. Scroll down to 'ORDER NOTES' section
4. Try "single tapping", to open the link or "double tapping", "pressing" to copy the note

**Screenshots**
![image](https://user-images.githubusercontent.com/506983/115810605-3303d380-a3bc-11eb-8a2c-1bde7bd18fb5.png)

![image](https://user-images.githubusercontent.com/506983/115810736-6e9e9d80-a3bc-11eb-9f55-8d2ce58a51f9.png)

**Expected behavior**
Either the link is clickable, or copyable.

**Isolating the problem (mark completed items with an [x]):**
- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**Mobile Environment**
Please include:
- Device: iPhone 11 Plus
- iOS version: 14.4.2
- WooCommerce iOS version: 6.3.0.2

**WordPress Environment**
<details>
```
`
### WordPress Environment ###

WordPress address (URL): https://breezecome.com
Site address (URL): https://breezecome.com
WC Version: 4.3.0
REST API Version: ✔ 1.0.10
WC Blocks Version: ✔ 2.7.1
Action Scheduler Version: ✔ 3.1.6
WC Admin Version: ✔ 1.3.0
Log Directory Writable: ✔
WP Version: ❌ 5.6.3 - There is a newer version of WordPress available (5.7.1)
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: –
WP Cron: ✔
Language: en_CA
External object cache: –

### Server Environment ###

Server Info: nginx/1.18.0
PHP Version: 7.4.3
PHP Post Max Size: 8 MB
PHP Time Limit: 0
PHP Max Input Vars: 1000
cURL Version: 7.68.0
OpenSSL/1.1.1f

SUHOSIN Installed: –
MySQL Version: 8.0.23-0ubuntu0.20.04.1
Max Upload Size: 2 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 4.3.0
WC Database Prefix: wp_
Total Database Size: 182.64MB
Database Data Size: 144.27MB
Database Index Size: 38.37MB
wp_woocommerce_sessions: Data: 0.08MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_order_items: Data: 0.08MB + Index: 0.05MB + Engine InnoDB
wp_woocommerce_order_itemmeta: Data: 0.52MB + Index: 0.52MB + Engine InnoDB
wp_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
core_store: Data: 0.06MB + Index: 0.00MB + Engine InnoDB
strapi_administrator: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
strapi_permission: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
strapi_role: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
strapi_users_roles: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
strapi_webhooks: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
upload_file: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
upload_file_morph: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
users-permissions_permission: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
users-permissions_role: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
users-permissions_user: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_actionscheduler_actions: Data: 27.06MB + Index: 16.13MB + Engine InnoDB
wp_actionscheduler_claims: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_groups: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_logs: Data: 11.03MB + Index: 9.03MB + Engine InnoDB
wp_affiliate_wp_affiliatemeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_affiliate_wp_affiliates: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_affiliate_wp_campaigns: Data: 0.00MB + Index: 0.00MB + Engine
wp_affiliate_wp_creatives: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_affiliate_wp_customermeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_affiliate_wp_customers: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_affiliate_wp_payouts: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_affiliate_wp_referralmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_affiliate_wp_referrals: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_affiliate_wp_rest_consumers: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_affiliate_wp_sales: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_affiliate_wp_visits: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_aioseo_notifications: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_aioseo_posts: Data: 1.50MB + Index: 0.06MB + Engine InnoDB
wp_categorymeta: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_commentmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_comments: Data: 1.52MB + Index: 0.67MB + Engine InnoDB
wp_dokan_announcement: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_dokan_orders: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_dokan_refund: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_dokan_vendor_balance: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_dokan_withdraw: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_formmaker: Data: 0.06MB + Index: 0.00MB + Engine InnoDB
wp_formmaker_backup: Data: 0.08MB + Index: 0.00MB + Engine InnoDB
wp_formmaker_blocked: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_formmaker_display_options: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_formmaker_groups: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_formmaker_query: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_formmaker_sessions: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_formmaker_submits: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_formmaker_themes: Data: 0.14MB + Index: 0.00MB + Engine InnoDB
wp_formmaker_views: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_grp_google_place: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_grp_google_review: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_imagify_files: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
wp_imagify_folders: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_links: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_member_catmeta: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_myCRED_log: Data: 0.14MB + Index: 0.02MB + Engine InnoDB
wp_nextend2_image_storage: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_nextend2_section_storage: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_nextend2_smartslider3_generators: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_nextend2_smartslider3_sliders: Data: 0.11MB + Index: 0.03MB + Engine InnoDB
wp_nextend2_smartslider3_sliders_xref: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_nextend2_smartslider3_slides: Data: 0.17MB + Index: 0.11MB + Engine InnoDB
wp_nf3_action_meta: Data: 0.05MB + Index: 0.00MB + Engine InnoDB
wp_nf3_actions: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_nf3_chunks: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_nf3_field_meta: Data: 1.52MB + Index: 0.00MB + Engine InnoDB
wp_nf3_fields: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_nf3_form_meta: Data: 0.05MB + Index: 0.00MB + Engine InnoDB
wp_nf3_forms: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_nf3_object_meta: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_nf3_objects: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_nf3_relationships: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_nf3_upgrades: Data: 0.48MB + Index: 0.00MB + Engine InnoDB
wp_ninja_table_items: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_options: Data: 11.13MB + Index: 0.41MB + Engine InnoDB
wp_pmxe_exports: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_pmxe_google_cats: Data: 0.39MB + Index: 0.00MB + Engine InnoDB
wp_pmxe_posts: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_pmxe_templates: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_portfolio_catmeta: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_postmeta: Data: 57.33MB + Index: 5.77MB + Engine InnoDB
wp_posts: Data: 21.30MB + Index: 0.70MB + Engine InnoDB
wp_product_catmeta: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_pts_modules: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_pts_modules_type: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_pts_tables: Data: 0.31MB + Index: 0.00MB + Engine InnoDB
wp_pts_usage_stat: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_responsive_menu: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_revslider_css: Data: 0.13MB + Index: 0.00MB + Engine InnoDB
wp_revslider_css_bkp: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_revslider_layer_animations: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_revslider_layer_animations_bkp: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_revslider_navigations: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_revslider_navigations_bkp: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_revslider_sliders: Data: 0.11MB + Index: 0.00MB + Engine InnoDB
wp_revslider_sliders_bkp: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_revslider_slides: Data: 1.45MB + Index: 0.00MB + Engine InnoDB
wp_revslider_slides_bkp: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_revslider_static_slides: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_revslider_static_slides_bkp: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_term_relationships: Data: 0.13MB + Index: 0.09MB + Engine InnoDB
wp_term_taxonomy: Data: 0.14MB + Index: 0.16MB + Engine InnoDB
wp_termmeta: Data: 0.05MB + Index: 0.03MB + Engine InnoDB
wp_terms: Data: 0.11MB + Index: 0.13MB + Engine InnoDB
wp_totalsoft_new_plugin: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_totalsoft_portfolio_albums: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_totalsoft_portfolio_dbt: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_totalsoft_portfolio_dbt_1: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_totalsoft_portfolio_dbt_2: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_totalsoft_portfolio_dbt_3: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_totalsoft_portfolio_dbt_4: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_totalsoft_portfolio_id: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_totalsoft_portfolio_images: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_totalsoft_portfolio_manager: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_usermeta: Data: 2.47MB + Index: 2.02MB + Engine InnoDB
wp_users: Data: 0.09MB + Index: 0.05MB + Engine InnoDB
wp_wc_admin_note_actions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_admin_notes: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_category_lookup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_customer_lookup: Data: 0.08MB + Index: 0.03MB + Engine InnoDB
wp_wc_download_log: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_coupon_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_product_lookup: Data: 0.13MB + Index: 0.06MB + Engine InnoDB
wp_wc_order_stats: Data: 0.08MB + Index: 0.05MB + Engine InnoDB
wp_wc_order_tax_lookup: Data: 0.06MB + Index: 0.03MB + Engine InnoDB
wp_wc_product_meta_lookup: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_wc_reserved_stock: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_tax_rate_classes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_webhooks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wpfb_post_templates: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wpfb_reviews: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wpforms_tasks_meta: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpmailsmtp_tasks_meta: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wysija_campaign: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wysija_campaign_list: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wysija_custom_field: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wysija_email: Data: 0.05MB + Index: 0.00MB + Engine InnoDB
wp_wysija_email_user_stat: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wysija_email_user_url: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wysija_form: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wysija_list: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wysija_queue: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wysija_subscriber_ips: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wysija_url: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wysija_url_mail: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wysija_user: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wysija_user_field: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wysija_user_history: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wysija_user_list: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_yith_wcwl: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_yith_wcwl_lists: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_yoast_indexable: Data: 1.30MB + Index: 0.48MB + Engine InnoDB
wp_yoast_indexable_hierarchy: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_yoast_migrations: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_yoast_primary_term: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_yoast_seo_links: Data: 0.22MB + Index: 0.14MB + Engine InnoDB

### Post Type Counts ###

attachment: 853
block: 26
custom_css: 5
customize_changeset: 1
elementor_library: 113
elementor-hf: 13
elementskit_content: 2
elementskit_template: 1
envato_tk_import: 9
fmemailverification: 1
foogallery: 10
form-maker: 1
ml-slide: 1
ml-slider: 1
mycred_badge: 1
mycred_email_notice: 3
mycred_rank: 4
nav_menu_item: 433
ninja-table: 1
oembed_cache: 1
page: 412
polylang_mo: 10
post: 37
product: 156
product_variation: 26
revision: 1283
rpt_pricing_table: 1
shop_coupon: 38
shop_order: 456
shop_order_refund: 4
tablepress_table: 1
vc_settings_preset: 1
viwec_template: 14
wpcf7_contact_form: 10
wpforms: 4
wysijap: 1

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (57) ###

Polylang: by WP SYNTEX – 2.9.1
All-in-One WP Migration: by ServMask – 7.25
Blog Designer Pack: by InfornWeb – 2.2.4
PublishPress Capabilities: by PublishPress – 1.10
CC Child Pages: by Caterham Computing – 1.39
Contact Form 7: by Takayuki Miyoshi – 5.2
Responsive Pricing Table: by WP Darko – 5.1.1
Duplicate Page: by mndpsingh287 – 4.4
Dynamic Featured Image: by Ankit Pokhrel – 3.7.0
Easy Table of Contents: by Steven A. Zahm – 2.0.11
Elementor: by Elementor.com – 2.9.14
ElementsKit Lite: by Wpmet – 1.5.10
Envato Elements: by Envato – 2.0.5
FooGallery: by FooPlugins – 1.9.31
Gallery Portfolio: by Total-Soft – 1.3.9
Google Analytics for WordPress by MonsterInsights: by MonsterInsights – 7.11.0
Elementor - Header, Footer & Blocks: by Brainstorm Force
Nikhil Chavan – 1.5.1

HT Mega - Absolute Addons for Elementor Page Builder: by HasThemes – 1.4.9
Imagify: by WP Media – 1.9.14
Social Slider Widget: by creativemotion – 1.7.9
Jetpack by WordPress.com: by Automattic – 9.6.1
WPBakery Page Builder: by Michael M - WPBakery.com – 6.2.0
Google Ads for WooCommerce: by Kliken – 1.0.4 – Installed version not tested with active version of WooCommerce 4.3.0
HubSpot for WooCommerce: by MakeWebBetter – 1.0.6
Multiple Roles: by Florian TIAR – 1.3.1
myCred: by myCred – 1.8.14.2
Ninja Forms: by Saturday Drive – 3.4.24.3
Ninja Tables: by WPManageNinja LLC – 4.0.2
Preloader Plus: by Massimo Sanfelice | Maxsdesign – 2.1.2
Pricing Table by Supsystic: by supsystic.com – 1.8.8
Profile Builder: by Cozmoslabs – 3.2.1
Really Simple SSL: by Really Simple Plugins – 3.3.4
Regenerate Thumbnails: by Alex Mills (Viper007Bond) – 3.1.3
Responsive Menu: by ExpressTech – 4.0.0
Slider Revolution: by ThemePunch – 6.2.2
Shopkeeper Extender: by GetBowtied – 1.6.0
Shopkeeper Portfolio Addon: by GetBowtied – 1.2.7
Simple Tags: by Amaury BALMER – 2.5.7
Smart Slider 3: by Nextend – 3.4.1.8
Sticky Header Effects for Elementor: by Rwattner – 1.3.2
TablePress: by Tobias Bäthge – 1.12
Tawk.to Live Chat: by Tawkto – 0.5.3
User Menus: by Code Atlantic – 1.2.4
Visual Portfolio: by nK – 2.2.0
Widget for Google Reviews: by RichPlugins  – 1.8.9
WooSwatches - Woocommerce Color or Image Variation Swatches: by woomatrix – 2.8.7 – Installed version not tested with active version of WooCommerce 4.3.0
WooCommerce Sam: by Sam Corp – 2.0.460 – Installed version not tested with active version of WooCommerce 4.3.0
WooCommerce: by Automattic – 4.3.0 (update to version 5.2.2 is available)
WordPress Importer: by wordpressdotorg – 0.7
Yoast SEO: by Team Yoast – 15.8
WP All Export: by Soflyy – 1.2.7
WP Google Review Slider: by LJ Apps – 7.4
WP HTML Mail - Email Template Designer: by Hannes Etzelstorfer // codemiq – 3.0.5
WP Mail SMTP: by WPForms – 2.2.1
WP Sitemap Page: by Tony Archambeau – 1.6.2
WPForms Lite: by WPForms – 1.6.4
Sequential Order Numbers for WooCommerce: by WebToffee – 1.2.3

### Inactive Plugins (35) ###

AffiliateWP: by Sandhills Development
LLC – 2.5.4

AffiliateWP - Affiliate Product Rates: by Pippin Williamson and Andrew Munro – 1.0.5.1
AffiliateWP - Sign Up Bonus: by Pippin Williamson and Andrew Munro – 1.1
Akismet Anti-Spam: by Automattic – 4.1.6
All in One SEO: by All in One SEO Team – 4.0.16
AMP: by AMP Project Contributors – 1.5.5
Bitnami Production Console Helper: by Bitnami – 1.2
Call Now Button: by Jerry Rietveld – 0.4.0
Customer Referral Program | Refer a Friend Software: by inviteReferrals – 2.2
Database Browser: by Chris Taylor – 1.4.3
Dokan: by weDevs – 3.0.7
Envato Market: by Envato – 2.0.3
Facebook for WooCommerce: by Facebook – 1.11.4 – Installed version not tested with active version of WooCommerce 4.3.0
Form Maker: by 10Web Form Builder Team – 1.13.40
Google Places Reviews: by Impress.org – 1.5.2
Happy Elementor Addons: by weDevs – 2.12.3
Hello Dolly: by Matt Mullenweg – 1.7.2
HookMeUp – Additional Content for WooCommerce: by GetBowtied – 1.3.0 – Installed version not tested with active version of WooCommerce 4.3.0
Kadence WooCommerce Email Designer: by Kadence WP – 1.4.7
Mailchimp for WooCommerce: by Mailchimp – 2.4.1 – Installed version not tested with active version of WooCommerce 4.3.0
MailPoet 2: by MailPoet – 2.14
Maintenance: by WP Maintenance – 3.90
Max Mega Menu: by megamenu.com – 2.9.1
Merchandiser Extender: by GetBowtied – 1.3.8
Mobile Menu: by Rui Guerreiro – 2.8.1.4
myStickymenu: by Premio – 2.4.3
One Click Demo Import: by ProteusThemes – 2.5.2
Porto Theme - Functionality: by P-Themes – 1.7.2
ShiftNav - Responsive Mobile Menu: by Chris Mavricos
SevenSpark – 1.7.0.1

Sticky Menu (or Anything!) on Scroll: by WebFactory Ltd – 2.23
W3 Total Cache: by BoldGrid – 2.1.0
WP Maintenance Mode: by Designmodo – 2.2.4
YITH WooCommerce Ajax Product Filter: by YITH – 3.11.1 – Installed version not tested with active version of WooCommerce 4.3.0
YITH WooCommerce Ajax Search: by YITH – 1.7.9 – Installed version not tested with active version of WooCommerce 4.3.0
YITH WooCommerce Wishlist: by YITH – 3.0.11 – Installed version not tested with active version of WooCommerce 4.3.0

### Settings ###

API Enabled: –
Force SSL: –
Currency: CAD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: –

### WC Pages ###

Shop base: #379 - /shop-2/
Cart: ❌ Page not set
Checkout: #381 - /checkout-2/
My account: #382 - /my-account/
Terms and conditions: ❌ Page not set

### Theme ###

Name: Shopkeeper
Version: 2.9.27
Author URL: https://www.getbowtied.com/
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
WooCommerce Support: ✔

### Templates ###

Overrides: shopkeeper/woocommerce/archive-product.php
shopkeeper/woocommerce/checkout/form-coupon.php
shopkeeper/woocommerce/checkout/form-login.php
shopkeeper/woocommerce/content-product.php
shopkeeper/woocommerce/content-single-product.php
shopkeeper/woocommerce/global/quantity-input.php
shopkeeper/woocommerce/global/sidebar.php
shopkeeper/woocommerce/global/wrapper-end.php
shopkeeper/woocommerce/global/wrapper-start.php
shopkeeper/woocommerce/myaccount/form-login.php
shopkeeper/woocommerce/single-product/add-to-cart/simple.php
shopkeeper/woocommerce/single-product/product-image.php
shopkeeper/woocommerce/single-product/product-thumbnails.php
shopkeeper/woocommerce/single-product/tabs/tabs.php
shopkeeper/woocommerce/single-product.php


### Action Scheduler ###

Complete: 47,953
Oldest: 2021-03-23 15:47:03 -0400
Newest: 2021-04-22 22:46:26 -0400

Failed: 7
Oldest: 2020-11-20 11:54:56 -0500
Newest: 2021-01-19 16:55:05 -0500

Pending: 12
Oldest: 2021-04-22 22:48:26 -0400
Newest: 2021-04-25 15:28:10 -0400

`

```
</details>

**Comment by pmusolino** [comment]
Thank you @maximmai for reporting this issue you encountered. It is definitely an improvement we can make to improve the user experience. I'll open a similar issue on Android.

**Comment by rachelmcr** [comment]
We've added the ability to copy notes in https://github.com/woocommerce/woocommerce-ios/pull/6894, released in version 9.3. Thanks for this suggestion!
